commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -67,7 +67,7 @@ def wrap(cls, data):
 
 
 class Settings(StrictJsonObject):
-    users = jsonobject.StringProperty(default='dimagi')
+    users = jsonobject.ListProperty(unicode, default='dimagi', required=True)
```

Comment:
does the default value also need to be a list or it's ok to leave as is?

Reply:
Good point. I don't see why the usernames shouldn't be specified so I removed the default entirely: https://github.com/dimagi/commcare-cloud/pull/1965/commits/6dfdf01e88cddac32b6e374b8884854e224494eb

Is that being used somewhere that I'm not seeing @dannyroberts ?